harness/helpers/helpers.ts

```diff
@@ -428,12 +428,15 @@ export class LocalTestClient extends BaseTestClient {
         return result
     }
 
-    getValidConfigPath() {
+    getValidConfigPath(sendV2 = false) {
+        if(sendV2) {
+            return `/client/${this.clientId}/config/v2/server/${this.sdkKey}.json`
+        } 
```

Comment:
yeah - just do the check for the capability here - you're already checking for it below

Reply:
fixed